libs/pdi-scanner/build.rs

```diff
@@ -0,0 +1,13 @@
+fn main() {
+    println!("cargo::rustc-check-cfg=cfg(production)");
+    println!("cargo:rerun-if-env-changed=NODE_ENV");
+
+    let is_production = match std::env::var("NODE_ENV") {
+        Ok(env) => env == "production",
+        Err(_) => false,
+    };
```

Comment:
One thing I'm unsure about is whether we also need to consider whether this is a VxDev setup vs. true prod. On VxDev, `NODE_ENV` is `production`, but `REACT_APP_VX_DEV` is set to `true` so that we can distinguish it from true prod, e.g.

https://github.com/votingworks/vxsuite/blob/0d0e272ba8aab6e61f5c43d5dcdc875b423a1738/libs/auth/src/config.ts#L24-L26

My quick sense is that, since VxDev isn't using VMs, it doesn't need the smaller buffer.

So all probably fine as is, just wanted to flag

Reply:
Let's do the simpler version to start. If it turns out VxDev doesn't work, we can update it.

Reply:
Yup sounds right to me!